scripts/update-command.py

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: MIT
+
+"""
+A Python script to update the common part of a command on all languages.
+
+Usage:
+    python3 scripts/update-command.py [-c] [-u] [-n] PLATFORM FILENAME [COMMAND]
+
+Options:
+    -c, --common-part COMMON_PART
+        Specify the common part to be modified (any content between double brackets will be ignored).
+    -u, --updated-common-part UPDATED_COMMON_PART
+        Specify the updated common part (any content between double brackets will be ignored).
+    -n, --dry-run
+        Show what changes would be made without actually modifying the page.
+
+
+Examples:
+    1. Update 'cargo' page interactively:
+       python3 scripts/update-command.py common cargo
+
+    2. Show what changes would be made by updating `sudo apt install {{}}` in 'apt' page to `sudo apt install {{}} --no-confirm`:
+       python3 scripts/update-command.py --dry-run -c `sudo apt install {{}}` -u `sudo apt install {{}} --no-confirm` linux apt
```

Comment:
```suggestion
       python3 scripts/update-command.py --dry-run -c "sudo apt install {{}}" -u "sudo apt install {{}} --no-confirm" linux apt
```

If not enclosed in proper quotes, the invoke the command modified too (or it exists before):

```zsh
(venv) ➜  tldr git:(tooling-update-command) ✗ python3 scripts/update-command.py --dry-run -c `sudo apt install {{}}` -u `sudo apt install {{}} --no-confirm` linux apt

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

E: Unable to locate package {{}}
E: Command line option --no-confirm is not understood in combination with the other options
usage: update-command.py [-h] [-c COMMON_PART] [-u UPDATED_COMMON_PART] [-n] [-v] platform filename
update-command.py: error: unrecognized arguments: Building dependency tree... Reading state information... apt
```

```zsh
(venv) ➜  tldr git:(tooling-update-command) ✗ python3 scripts/update-command.py --dry-run -c 'sudo apt install {{}}' -u 'sudo apt install {{}} --no-confirm' linux apt
Enter the commands (any content between double curly brackets will be ignored):
```

This is with double quotes:

```zsh
(venv) ➜  tldr git:(tooling-update-command) python3 scripts/update-command.py --dry-run -c "sudo apt install {{}}" -u "sudo apt install {{}} --no-confirm" linux apt
Enter the commands (any content between double curly brackets will be ignored):
`sudo apt install {{package}}` -> `sudo apt install {{package}} --no-confirm`
`sudo apt install {{paquet}}` -> `sudo apt install {{paquet}} --no-confirm`
`sudo apt install {{package}}` -> `sudo apt install {{package}} --no-confirm`
`sudo apt install {{pakket1 pakket2 ...}}` -> `sudo apt install {{pakket1 pakket2 ...}} --no-confirm`
`sudo apt install {{paket}}` -> `sudo apt install {{paket}} --no-confirm`
`sudo apt install {{الحزمة}}` -> `sudo apt install {{الحزمة}} --no-confirm`
`sudo apt install {{package}}` -> `sudo apt install {{package}} --no-confirm`
`sudo apt install {{pacchetto}}` -> `sudo apt install {{pacchetto}} --no-confirm`
`sudo apt install {{പാക്കേജ്}}` -> `sudo apt install {{പാക്കേജ്}} --no-confirm`
`sudo apt install {{パッケージ}}` -> `sudo apt install {{パッケージ}} --no-confirm`
`sudo apt install {{пакет}}` -> `sudo apt install {{пакет}} --no-confirm`
`sudo apt install {{nome_do_pacote}}` -> `sudo apt install {{nome_do_pacote}} --no-confirm`
`sudo apt install {{软件包}}` -> `sudo apt install {{软件包}} --no-confirm`
`sudo apt install {{paket}}` -> `sudo apt install {{paket}} --no-confirm`
`sudo apt install {{पैकेज}}` -> `sudo apt install {{पैकेज}} --no-confirm`
`sudo apt install {{প্যাকেজ}}` -> `sudo apt install {{প্যাকেজ}} --no-confirm`
`sudo apt install {{nome_do_pacote}}` -> `sudo apt install {{nome_do_pacote}} --no-confirm`
`sudo apt install {{paquete}}` -> `sudo apt install {{paquete}} --no-confirm`
`sudo apt install {{pakiet}}` -> `sudo apt install {{pakiet}} --no-confirm`
`sudo apt install {{நிரல்தொகுப்பு}}` -> `sudo apt install {{நிரல்தொகுப்பு}} --no-confirm`
`sudo apt install {{nama_paket}}` -> `sudo apt install {{nama_paket}} --no-confirm`
```